src/index.html

```diff
@@ -8,6 +8,13 @@
   <link rel="stylesheet" href="./styles/main.css">
 </head>
 <body>
-  <h1>Antisnake</h1>
+  <div class="anti-snake container">
```

Comment:
don't give two classes, remove .container from html & css
```suggestion
  <div class="anti-snake">
```